backend/onyx/document_index/vespa/index.py

```diff
@@ -347,9 +348,9 @@ def index(
                     executor=executor,
                 )
 
-            # c) Remove chunks with using versioning scheme 'last_indexed_at'
+            # c) Remove chunks with using versioning scheme 'current_index_time'
```

Comment:
can deleting old chunks after feeding new chunks cause any issues?  As in returning out of date chunks in results?

Reply:
Yes I believe so, although the old approach had issues too (e.g. post deletion, pre re-insertion doc would not show up). Hard to say which is preferable.